build.cmd

```diff
@@ -28,27 +32,48 @@ if '!_FloatingDependencies!'=='0' (
   set _defaultBuildConfig=/p:Configuration=Windows_NT_Debug
   )
 )
+:: *** end WCF Content ***
+
+:ReadArguments
+:: Read in the args to determine whether to run the native build, managed build, or both (default)
+set "__args=%*"
+if /i [%1] == [native] (set __buildSpec=native&&set "__args=%__args:~6%"&&shift&&goto Tools)
```

Comment:
True we don't have any native libraries but this is a conditional check right? We don't need a native build therefore we don't specify that argument therefore we don't do a native build?

How do you mean the script will bail?